src/main/java/exception/TestExc.java

```diff
@@ -0,0 +1,84 @@
+package exception;
+
+import main.Employee;
+import main.ExecutionType;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TestExc {
+    /**
+     * TASK #1
+     * Витягнути ліст імен, Видалити всіх Олегів з ліста (відповідно, щоб у лісті було їх хоча б кілька)
+     */
+    public List<String> removeEmployee(List<Employee> employeeList, ExecutionType executionType, String targetName) throws NoSuchEmployeeException {
```

Comment:
NoSuchEmployeeException - is Runtime Exception. So there is no need to add it after method signature